ivadomed/main.py

```diff
@@ -615,7 +618,7 @@ def cmd_test(context):
 
     print(f"Loaded {len(ds_test)} {context['slice_axis']} slices for the test set.")
     test_loader = DataLoader(ds_test, batch_size=context["batch_size"],
-                             shuffle=True, pin_memory=True,
+                             shuffle=False, pin_memory=True,
```

Comment:
`shuffle=False` --> in order to perform the prediction on consecutive slices, easier to further reconstruct the 3D vol.